server/src/models/Message.js

```diff
@@ -0,0 +1,37 @@
+const Model = require("./Model")
+
+class Message extends Model {
+    static get tableName() {
+        return "messages"
+    }
+
+    static get jsonSchema() {
+        return {
+            type: "object",
+            required: ["content", "messageType", "dialogId"],
+            properties: {
+                content: { type: "string" },
+                messageType: { type: "string" },
+                reviewed: { type: "boolean" },
```

Comment:
depending on how you are managing this `reviewed` property, might need to allow type boolean and string